src/chains.ts

```diff
@@ -303,7 +303,7 @@ export const AVALANCHE_MAINNET: Chain = {
   supported_by_twap: true,
   rpc_uri: 'https://api.avax.network/ext/bc/C/rpc',
   subgraph: {
-    exchange: 'https://api.thegraph.com/subgraphs/name/pangolindex/exchange'
+    exchange: 'https://api.thegraph.com/subgraphs/name/sarjuhansaliya/minichefv2-dummy'
```

Comment:
@SarjuHansaliya from my understanding there are two subgraphs. The Exchange Subgraph and then the new one that @bmino deployed which gets information for SSS. So we'll probably need to add another field here.

Reply:
@HariSeldon23 yes you are right. this field is not being used anywhere at the moment. but I will add `exchange`, `minichef`, `pangochef`, `blocks` fields under `subgraph`